oximeter/db/Cargo.toml

```diff
@@ -16,19 +16,24 @@ bcs.workspace = true
 camino.workspace = true
 chrono.workspace = true
 clap.workspace = true
+crossterm.workspace = true
```

Comment:
The changes in this file don't look right to me. Crates like `crossterm` are still optional, and only needed if the `sql` feature is enabled, unless I'm missing something?

Reply:
I'll double-check. I thought it was on oxql too, but can remove.

Reply:
Ah, the Q is if oxql should remain featured flagged, even if it remains in oxdb (but is still called out via the mod in omdb)? Feels not right to feature flag it if it's expected in the mainline bin?

Reply:
Yeah, I was suggesting we still have the `oxql` feature flag, which would mean these crates are still optional. I've generally tried to provide feature-gates for large and / or uncertain, independent chunks of code, even if they're on by default. But I don't have a strong opinion at this point.

Reply:
@bnaecker makes sense. I'll bring it back and foward it through now that the shell run command is available in the lib piece of omdb.